recipes-extended/xen/xen-hypervisor.bb

```diff
@@ -87,7 +90,7 @@ do_compile() {
     export CPP="${HOST_PREFIX}cpp ${TOOLCHAIN_OPTIONS}"
 
     oe_runmake -C xen olddefconfig
-    # public/v4v.h is not compliant at all, headers++.chk will fail trying as
+    # public/argo.h is not compliant at all, headers++.chk will fail trying as
```

Comment:
Is this true? I don't think it is and the workaround isn't needed in meta-virtualization, so can this CXX override be dropped?

Reply:
@dozylynx It was for `v4v.h`, it is no longer the case with `argo.h`. This can be dropped.